### PR TITLE
Upgrade to python 3.7.5

### PR DESCRIPTION
**Pull request by Ram81**
### Description

This PR adds -
- [x] Upgrade django and worker container python to 3.7.5
- [x] Upgrade django and worker container pip to 20.2.3

### CODE REVIEW

File: docker/dev/django/Dockerfile [hunk 1]
```diff
@@ -7,6 +7,7 @@ WORKDIR /code
 
 ADD requirements/* /code/
 
+RUN pip install --upgrade pip==20.2.3
```
Comment (RishabhJain2018) [inline docker/dev/django/Dockerfile:10]: Let's update it to `19.x` as we discussed.

File: docker/prod/django/Dockerfile [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+FROM python:3.7.5
 
 ENV PYTHONUNBUFFERED 1
 
 RUN mkdir /code
 WORKDIR /code
+
 ADD requirements/* /code/
+
+RUN pip install --upgrade pip==20.2.3
```
Comment (RishabhJain2018) [inline docker/prod/django/Dockerfile:10]: Same as above.

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]
Can you please add the same change in the Travis config.

**Comment by Ram81** [comment]
@RishabhJain2018  which change in travis? it uses same docker build. Do you meant the pip change?

**Comment by RishabhJain2018** [comment]
No, the python version change.

File: .travis.yml [hunk 3]
```diff
@@ -1,4 +1,6 @@
 language: python
+python:
```
Comment (RishabhJain2018) [inline .travis.yml:2]: There is already a `python` in this file with version `3.6` :) . Let's update that.

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]

File: docker/prod/django/Dockerfile [hunk 4]
```diff
@@ -1,10 +1,12 @@
-FROM python:3.6.5
+FROM python:3.7.5
 
 ENV PYTHONUNBUFFERED 1
 
 RUN mkdir /code
 WORKDIR /code
+
```
Comment (RishabhJain2018) [inline docker/prod/django/Dockerfile:7]: Please remove the blank line.

File: docker/prod/django/Dockerfile [hunk 5]
```diff
@@ -1,10 +1,12 @@
...
 ADD requirements/* /code/
+
```
Comment (RishabhJain2018) [inline docker/prod/django/Dockerfile:9]: Please remove blank line.

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**